core/deployment/src/main/java/io/quarkus/deployment/steps/BannerProcessor.java

```diff
@@ -38,8 +42,12 @@ private String readBannerFile(BannerConfig config) {
         if (resource != null) {
             try (InputStream is = resource.openStream()) {
                 byte[] content = FileUtil.readFileContents(is);
-                String bannerTitle = new String(content, StandardCharsets.UTF_8);
-                return bannerTitle + "\n:: Quarkus :: v" + Version.getVersion() + '\n';
+                StringBuilder bannerTitle = new StringBuilder(new String(content, StandardCharsets.UTF_8));
+                bannerTitle.append('\n');
+                if (!config.isDefaultPath()) {
+                    bannerTitle.append("Powered by Quarkus v").append(Version.getVersion()).append('\n');
```

Comment:
I got a PR on https://github.com/geoand/quarkus/pull/1 but for right padding but for some reason that gives me a NPE in the java compiler ;)